platforms/documentation/docs/src/docs/release/notes.md

```diff
@@ -52,7 +52,18 @@ Example:
 ADD RELEASE FEATURES BELOW
 vvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvv -->
 
+<a name="other"></a>
+### Other improvements
 
+#### Tests metadata improvements in tooling API
+
+IDEs and other tools can use tooling API to get information about tests Gradle runs.
+Each test event sent by tooling API contains a test descriptor that contains test metadata: human-readable name, class name, method name, etc.
+
+For JUnit5 and Spock, we updated the test descriptor for dynamic and parametrized tests to include the data about the class name and method name containing the test.
+We also improved the display names, so they are passed transparently from the frameworks and can be used by IDEs without transformations.
```

Comment:
```suggestion
Additionally, we improved the display names so that they are transparently passed from the frameworks, enabling IDEs to use them without requiring transformations.
```